addon/addon.json

```diff
@@ -1,7 +1,5 @@
 [
```

Comment:
Are you deleting them?

Reply:
Yeah, i did it on my side, as I didn’t have you updates before you committed a big update with that fix